tests/requirements-test.txt

```diff
@@ -1,6 +1,6 @@
 versioneer
 # pytest v3.0 (from 19.08.2016) is no longer compatible with Python 3.2.x
-pytest<3.0 ; python_version >= '3.2' and python_version < '3.3'
+pytest<3.0 ; python_version >= '3.2' and python_version < '3.3' # rq.filter: >=2.9.2,<3.0
```

Comment:
Could you please drop a line of comment explaining **why** this modification was done and what is rq.filter ?